cli/cmd/plugin/ui/web/tanzu-ui/src/shared/services/awsDeployment.tsx

```diff
@@ -66,14 +66,7 @@ const useAwsDeployment = () => {
             ceipOptIn: awsState[STORE_SECTION_FORM].ENABLE_CEIP_PARTICIPATION,
             labels: {},
             // TODO: define a default OS image to set via aws.defaults.tsx
-            os: {
-                name: 'ubuntu-20.04-amd64 (ami-0dd0327a3bfaa4dc8)',
-                osInfo: {
-                    arch: 'amd64',
-                    name: 'ubuntu',
-                    version: '20.04',
-                },
-            },
+            os: awsState[STORE_SECTION_FORM].OS_IMAGE,
```

Comment:
I notice there is an AWS_FIELDS.IMAGE_INFO enum, which I'm wondering if it should be used here (`awsState[STORE_SECTION_FORM][AWS_FIELDS.IMAGE_INFO]`).

Reply:
Got it. Will do. Thanks

Reply:
I agree it would be best to change them all, but I don't want to hold up this PR on that account. :)